srcpkgs/protontricks/template

```diff
@@ -1,9 +1,9 @@
 # Template file for 'protontricks'
 pkgname=protontricks
-version=1.8.2
+version=1.9.0
 revision=1
 build_style=python3-module
-hostmakedepends="python3-setuptools_scm"
+hostmakedepends="python3-setuptools python3-setuptools_scm"
```

Comment:
this isn't necessary, python3-setuptools_scm depends on python3-setuptools already